src/main/interface/tramp_protocol.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "drivers/serial.h"
+
+#define TRAMP_SERIAL_OPTIONS            SERIAL_NOT_INVERTED | SERIAL_BIDIR
+#define TRAMP_BAUD                      9600
+#define TRAMP_PAYLOAD_LENGTH            12
+
+typedef struct trampSettings_s {
+    uint16_t frequency;
+    uint16_t power;
+    uint8_t raceModeEnabled;
+    uint8_t pitModeEnabled;
+} trampSettings_t;
+
+typedef struct trampFrameHeader_s {
```

Comment:
Turns out you already have it. Say no to magic defines then, define them in terms of struct sizes